tests/scripts/pds_request.py

```diff
@@ -2,7 +2,7 @@
 from json import loads, JSONDecodeError
 from uuid import uuid4
 from requests import Response
-from . import config
+from ..functional.configuration import config
```

Comment:
can this be converted to absolute import?

Reply:
Done.